cookbook/setup.md

```diff
@@ -95,11 +96,11 @@ To be most compatible, the `starship` binary will run every prompt render and
 is absolute stateless.
 Nushell, however, is very stateful in a single instance.
 
-[Hooks](https://www.nushell.sh/book/hooks.html#hooks) allow registration of
+[Hooks](../book/hooks.md) allow registration of
```

Comment:
It's better to use absolute path here: '/zh-CN/book/overlays.md'

Reply:
The absolute path is more intuitive and stable, but if relative paths are used elsewhere, I'm fine with keeping things as they are.